### ISSUE TITLE
Max exceeds maximum extrusion - just in orca

### ISSUE THREAD

**Issue by d9C89** [SoftFever/OrcaSlicer#2895]
Hello, I have a problem with the orcaslicer!

. I slice a part and always at exactly the same place it stops and spits out this error message. I only have that with the Orcaslicer. If I print with similar parameters with prusa, it's not a problem. So what do I have to set there?

![IMG-20231126-WA0007.jpg](https://github.com/SoftFever/OrcaSlicer/assets/129627541/6fbe8938-771c-4703-87d0-7403804c7754)

**Comment by mjfsch** [comment]
do you have any colour changes in this? i had the same issue with filament ramming - had to change max_extrude_cross_section in printer.cfg to allow for it.
If you have changed nozzle size then it may be related to that.

**Comment by d9C89** [comment]
> do you have any colour changes in this? i had the same issue with filament ramming - had to change max_extrude_cross_section in printer.cfg to allow for it. If you have changed nozzle size then it may be related to that.

no, i have an idex, use 2 filaments. sv04. the print starts, an always on the same spot(after ca. 1h) error

**Comment by StarkRG** [comment]
This sounds like the same bug as #1459
